README.md

```diff
@@ -120,14 +184,100 @@ This folder contains examples that illustrate the use of Amazon Braket Hybrid Jo
+
+* [**Running Jupyter notebooks as a Hybrid Job**](examples/hybrid_jobs/7_Running_notebooks_as_hybrid_jobs/Running_notebooks_as_hybrid_jobs.ipynb)
+
+    This tutorial is a step-by-step guide for running a Jupyter notebook as a Hybrid Job.
+
+* [**Creating Hybrid Job Scripts**](examples/hybrid_jobs/8_Creating_Hybrid_Job_Scripts/Creating_your_first_Hybrid_Job.ipynb)
+
+    This notebook shows an alternate way to create a Hybrid Job - without using a @hybrid_job decorator - that may be useful in some circumstances, such as using older versions of Python. 
+
+---
+## <a name="pulse">Pulse Control</a>
+
+* [**Bringup Experiments**](examples/pulse_control/1_Bringup_experiments.ipynb)
+
+    This tutorial will introduce common pulse sequences and calibrating pulses via Rabi spectroscopy.
+
+* [**Native Gate Calibrations**](examples/pulse_control/2_Native_gate_calibrations.ipynb)
+
+    This tutorial shows how to retrieve the calibrations of native gates for Rigetti's Aspen devices and submit a circuit with custom gate calibrations.
+
+* [**Bell pair with pulses (OQC)**](examples/pulse_control/3_Bell_pair_with_pulses_OQC.ipynb)
+
+    This tutorial shows creating a Bell state with cross-resonance pulses on OQC’s Lucy device.
+
+* [**Bell pair with pulses (Rigetti)**](examples/pulse_control/4_Bell_pair_with_pulses_Rigetti.ipynb)
+
+    This tutorial shows creating a Bell state with cross-resonance pulses on Rigetti's Aspen device.
+
+* [**Build single qubit gates**](examples/pulse_control/5_Build_single_qubit_gates.ipynb)
+
+    This tutorial describes a method to create any single-qubit gate with pulses.
+
+---
+## <a name="ahs">Analog Hamiltonian Simulation</a>
+
+* [**Intruction to Analog Hamiltonian Simulation**](examples/analog_hamiltonian_simulation/00_Introduction_of_Analog_Hamiltonian_Simulation_with_Rydberg_Atoms.ipynb)
+
+    This tutorial provides an introduction to Analog Hamiltonian Simulation (AHS), a quantum computing paradigm different from gate-based computing. AHS uses a well-controlled quantum system and tunes its parameters to mimic the dynamics of another quantum system, the one we aim to study.
+
+* [**Intruction to Aquila**](examples/analog_hamiltonian_simulation/01_Introduction_to_Aquila.ipynb)
+
+    This tutorial illustrates how to run an AHS program on QuEra’s Aquila, a Rydberg based QPU, via Amazon Braket.
+
+* [**Ordered Phases in Rydberg Systems**](examples/analog_hamiltonian_simulation/02_Ordered_phases_in_Rydberg_systems.ipynb)
+
+    This tutorial shows how to prepare ordered phases in Rydberg systems, focusing on the 1D phase and the 2D checkerboard phase. It uses an adiabatic time-evolution to prepare these many-body ground states.
+
+* [**Parallel Tasks on Aquila**](examples/analog_hamiltonian_simulation/03_Parallel_tasks_on_Aquila.ipynb)
+
+    This tutorial goes through the previously explored checkerboard preparation but now takes advantage of the full area.
```

Comment:
I am worried this description falls out of sync and customers are left wondering, "What previously explored checkboard prep".

Reply:
It's referencing the bullet point above this one. I don't think it's much of a stretch.

Reply:
Reworded. Is this better, or worse?